core/src/main/java/com/scalar/db/service/TransactionFactory.java

```diff
@@ -26,6 +27,15 @@ public DistributedTransactionManager getTransactionManager() {
     return injector.getInstance(DistributedTransactionManager.class);
   }
 
+  /**
+   * Returns a {@link DistributedTransactionAdmin} instance
+   *
+   * @return a {@link DistributedTransactionAdmin} instance
+   */
+  public DistributedTransactionAdmin getTransactionAdmin() {
```

Comment:
We can get an instance of `DistributedTransactionAdmin` with this `TransactionFactory.getTransactionAdmin()`.